packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -10,6 +10,29 @@ import { getScrollParent } from './getScrollParent';
 import { getPlacement, applyRtlToOffset } from './positioningHelper';
 import { PopperModifiers, PopperProps } from './types';
 
+// https://github.com/facebook/react/blob/848bb2426e44606e0a55dfe44c7b3ece33772485/packages/react-dom/src/client/ReactDOMHostConfig.js#L157-L166
+const isAutofocusEnabled = (node: Node, reactInstanceKey: string): boolean => {
+  return (
+    (node.nodeName === 'BUTTON' ||
+      node.nodeName === 'INPUT' ||
+      node.nodeName === 'SELECT' ||
+      node.nodeName === 'TEXTAREA') &&
+    node[reactInstanceKey].pendingProps.autoFocus
+  );
+};
+
+const getReactInstanceKey = (elm: HTMLElement): string => {
+  if (!elm) return null;
+
+  for (const k in elm) {
+    if (k.startsWith('__reactInternalInstance$')) {
```

Comment:
`.startsWith()` is not in IE11 yet. We can replace it with `indexOf()` or the Lodash one.

`ReactInstanceKey` is also constant for the whole app lifecycle, we can cache into some variable:

```tsx
let reactInstanceKey: string

const getReactInstanceKey = (elm: HTMLElement): string => {
  if (!reactInstanceKey) {
     reactInstanceKey = ...
  }

  return reactInstanceKey
}
```